bmwqemu.pm

```diff
@@ -198,24 +197,24 @@ sub update_line_number {
 }
 
 # pretty print like Data::Dumper but without the "VAR1 = " prefix
-sub pp {
+sub pp(@) {
```

Comment:
This should be named and `@_` 2 lines later replaced with that.
At some point `@_` will be deprecated in signature subs.

Reply:
done